config/schema.yaml

```diff
@@ -0,0 +1,330 @@
+  doc: database_url
+  env: DATABASE_URL
+dev_mode:
+  default: true
+  doc: dev_mode
+  env: DEV_MODE
+disable_intercom:
+  default: true
+  doc: disable_intercom
+  env: DISABLE_INTERCOM
+disable_plans:
+  default: true
+  doc: disable_plans
+  env: DISABLE_PLANS
+domainWhitelist:
+  default:
+  - ^localhost$
+  - ^192\\.168\\.1\\.141$
+  - ^192\\.168\\.1\\.140$
+  - ^pol\\.is
+  - .+\\.pol\\.is$
+  - ^xip\\.io$
+  - .+\\.xip\\.io$
+  doc: domainWhitelist
+  env: DOMAIN_WHITELIST
+domain_override:
+  default: ''
+  doc: domain_override
+  env: DOMAIN_OVERRIDE
+domain_whitelist_item_01:
+  default: ''
+  doc: domain_whitelist_item_01
+  env: DOMAIN_WHITELIST_ITEM_01
+domain_whitelist_item_02:
+  default: ''
+  doc: domain_whitelist_item_02
+  env: DOMAIN_WHITELIST_ITEM_02
+domain_whitelist_item_03:
+  default: ''
+  doc: domain_whitelist_item_03
+  env: DOMAIN_WHITELIST_ITEM_03
+domain_whitelist_item_04:
+  default: ''
+  doc: domain_whitelist_item_04
+  env: DOMAIN_WHITELIST_ITEM_04
+domain_whitelist_item_05:
+  default: ''
+  doc: domain_whitelist_item_05
+  env: DOMAIN_WHITELIST_ITEM_05
+domain_whitelist_item_06:
+  default: ''
+  doc: domain_whitelist_item_06
+  env: DOMAIN_WHITELIST_ITEM_06
+domain_whitelist_item_07:
+  default: ''
+  doc: domain_whitelist_item_07
+  env: DOMAIN_WHITELIST_ITEM_07
+domain_whitelist_item_08:
+  default: ''
+  doc: domain_whitelist_item_08
+  env: DOMAIN_WHITELIST_ITEM_08
+email_transport_types:
+  default: maildev
+  doc:
+  - email_transport_types - maildev, aws-ses, mailgun
+  - ex. `aws-ses,mailgun` would try sending via AWS SES first, and fallback to Mailgun
+    on error.
+  - cypress-tests.yml uses - EMAIL_TRANSPORT_TYPES=mailgun,aws-ses,nonexistent,maildev
+  env: EMAIL_TRANSPORT_TYPES
+encryption_password_00001:
+  default: c9336642-2024-43a8-99cd-77ee38e82a9c
+  doc: encryption_password_00001
+  env: ENCRYPTION_PASSWORD_00001
+env:
+  default: development
+  doc: The application environment
+  env: NODE_ENV
+  format:
+  - production
+  - development
+  - test
+fb_app_id:
+  default: 661042417336977
+  doc: must register with facebook and get a facebook app id to use the facebook auth
+    features
+  env: FB_APP_ID
+git_hash:
+  default: undefined
+  doc: git_hash
+  env: GIT_HASH
+google_api_key:
+  default: undefined
+  doc: google_api_key
+  env: GOOGLE_API_KEY
+google_credentials_base64:
+  default: X
+  doc: google_credentials_base64
+  env: GOOGLE_CREDENTIALS_BASE64
+google_creds_stringified:
+  default: X
+  doc: google_creds_stringified
+  env: GOOGLE_CREDS_STRINGIFIED
+intercom_access_token:
+  default: undefined
+  doc: intercom_access_token
+  env: INTERCOM_ACCESS_TOKEN
+local_output_path:
+  default: ./build
+  doc: Uploader settings local
+  env: LOCAL_OUTPUT_PATH
+mailgun_api_key:
+  default: undefined
+  doc: mailgun_api_key
+  env: MAILGUN_API_KEY
+mailgun_domain:
+  default: undefined
+  doc: mailgun_domain
+  env: MAILGUN_DOMAIN
+math_env:
```

Comment:
We should probably default (or even just hard code) this to `prod`.

Context: Prior to us open sourcing and making deploys more repeatable, Polis was built around a single source of truth database, which we used for both staging ("`preprod`") and production (`prod`). Math results were tagged with one of these `math_env` variables so that the two systems could use the same database, and have access to the same underlying vote data. There was some convenience to this when Polis was closed source, but makes no sense now. Currently, even our "preprod" environment uses `math_env=prod`.